.ibm/pipelines/openshift-tests.sh

```diff
@@ -196,7 +196,7 @@ main() {
 
   echo "Tag name with short SHA: pr-${GIT_PR_NUMBER}-${SHORT_SHA}"
 
-  helm upgrade -i ${RELEASE_NAME} -n ${NAME_SPACE} ${HELM_REPO_NAME}/${HELM_IMAGE_NAME} --version ${CHART_VERSION} -f $DIR/value_files/${HELM_CHART_VALUE_FILE_NAME} --set global.clusterRouterBase=${K8S_CLUSTER_ROUTER_BASE} --set upstream.backstage.image.tag=pr-${GIT_PR_NUMBER}-${SHORT_SHA}
+  helm upgrade -i ${RELEASE_NAME} -n ${NAME_SPACE} rhdh-chart/redhat-developer-hub --version ${CHART_VERSION} -f $DIR/value_files/${HELM_CHART_VALUE_FILE_NAME} --set global.clusterRouterBase=${K8S_CLUSTER_ROUTER_BASE} --set upstream.backstage.image.tag=pr-${GIT_PR_NUMBER}-${SHORT_SHA}
```

Comment:
are we sure this is correct? 

looking at https://artifacthub.io/packages/helm/rhdh-chart/backstage I see the install instructions are:

```
helm install my-backstage oci://ghcr.io/redhat-developer/rhdh-chart/backstage --version 2.13.2
```